src/App.js

```diff
@@ -1,41 +1,21 @@
-import "./App.css";
-import logo from "./logo.png";
-
+import './App.css';
+import Navbar from './components/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import Home from './pages/Home';
```

Comment:
Optional - You could import all the components into one components.jsx and then import components from there,like this -
https://codesandbox.io/s/usereducer-infinity-wars-final-7n11ox?file=/src/utils/index.jsx